mailbox/api/src/main/java/org/apache/james/mailbox/model/MessageResult.java

```diff
@@ -244,6 +247,13 @@
     Headers getHeaders() throws MailboxException;
     
     /**
+     * Return the list of attachments
+     * 
+     * @return a read only list of attachments
```

Comment:
you prefer I remove the comment or that I change the method signature?
Usually we are returning List everywhere even if we are always returning Immutable ones.